ste/sender-azureFile.go

```diff
@@ -446,7 +446,8 @@ func (d AzureFileParentDirCreator) CreateDirToRoot(ctx context.Context, dirURL a
 					// Must do it here, in the routine that is shared by both the folder and the file code,
 					// because due to the parallelism of AzCopy, we don't know which will get here first, file code, or folder code.
 					dirUrl := curDirURL.URL()
-					t.RecordCreation(dirUrl.String())
+					dirUrlStr := strings.Replace(dirUrl.String(), ";", url.QueryEscape(";"), -1)
+					t.RecordCreation(dirUrlStr)
```

Comment:
Is `;` the only one of these "unsafe" characters? Should we instead use the encoding rules from the `cmd` package?

Reply:
I noticed that `url.String()` method, which we have in file SDK, doesn't encode `;` but encode other dir/file with special characters. I tried that with general case and it seems to work.